pkgs/top-level/all-packages.nix

```diff
@@ -15357,8 +15357,6 @@ with pkgs;
 
   pgcli = pkgs.python3Packages.pgcli;
 
-  phantomjs = callPackage ../development/tools/phantomjs { };
-
   phantomjs2 = libsForQt514.callPackage ../development/tools/phantomjs2 { };
```

Comment:
how about phantomjs2? 
the npm site says its been deprecated
https://www.npmjs.com/package/phantomjs2

Reply:
Yes, PhantomJS development stopped a while ago (https://github.com/ariya/phantomjs/issues/15344). But the last 2.x version might still be useful for some :shrug: